packages/web/src/pages/chat-page/components/ChatBlastCTA.tsx

```diff
@@ -35,8 +35,7 @@ export const ChatBlastCTA = (props: ChatBlastCTAProps) => {
   const { data: userId } = useGetCurrentUserId({})
   const { tierNumber, isVerified } = useSelectTierInfo(userId ?? 0) ?? {}
   const chatBlastTier = useRemoteVar(IntKeys.CHAT_BLAST_TIER_REQUIREMENT)
-  const userMeetsRequirements =
-    isVerified || (tierNumber && tierNumber >= chatBlastTier)
+  const userMeetsRequirements = isVerified || tierNumber >= chatBlastTier
```

Comment:
yeah `undefined` or `null` will fail the `>=` check so lgtm

Reply:
turns out `null >= 0` is `true` (thanks js) but `tierNumber` is of type `number` so i'm gonna trust that. we also have a ticket to restrict blast access on protocol side so if this check fails we'll still check there.